src/index.html

```diff
@@ -2,16 +2,193 @@
+            type="date"
+            name="dob"
+            id="dob"
+            required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div class="form-group">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            name="email"
+            required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-group">
+          <label>Do you love cats?</label>
+          <input
+            type="radio"
+            name="cats"
+            value="yes"
+            id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            value="no"
+            id="no"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="form-group">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+            required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="time"
+            name="time"
+            step="2"
+            required
+          >
+        </div>
+
+        <div class="form-group">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+            id="cars"
+            name="cars"
+            multiple
+            required
+          >
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+          </select>
+        </div>
+
+        <div class="form-group">
+          <label for="work">How do you rate our work?</label>
+          <input
+            type="range"
+            id="work"
+            name="work"
+            value="0"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+
+        <div class="form-group">
+          <label for="comments">Comments:</label>
+          <textarea
+            name="comments"
+            id="comments"
+            cols="20"
+            rows="2"
+            required
+          ></textarea>
```

Comment:
let's fix this error
![image](https://user-images.githubusercontent.com/102293862/214045169-3acff67a-9f77-4eaa-a55d-c8af656e23fd.png)

Reply:
I am not getting how to solve that space in textarea